draft-ietf-scitt-architecture.md

```diff
@@ -201,7 +201,7 @@ Artifact:
 
 Auditor:
 
-: an entity that checks the correctness and consistency of all Transparent Statements registered by a Transparency Service (a subset of potential Transparent Statement Consumers).
+: an entity that checks the correctness and consistency of all Transparent Statements issued by a Transparency Service (to a subset of potential Transparent Statement Consumers).
```

Comment:
```suggestion
: an entity that checks the correctness and consistency of all Transparent Statements issued by a Transparency Service.
```

Reply:
"issued to" will lead to the 2 party vs 3 party model discussion... we probably don't want to invite that.

Reply:
better?

Reply:
Done the change in the new commit